css/stylesheet.css

```diff
@@ -0,0 +1,125 @@
+/*
+
+{font-family: Oswald}
+{font-family: Lato}
+{font-family: Merriweather}
+{font-family: Lora}
+*/
+
+.main-body {
+  margin-left: 12.5%;
+  margin-right: 12.5%;
+}
+
+.l-masthead {
+  margin-top:50px;;
+
+}
+.l-box-model {
+  width: 35%;
+  float: left;
+  margin-left: -10%;;
+
+}
+.l-feature {
+  width: 65%;
+  float: right;
+}
+.l-staying-afloat {
+  text-align:right;
```

Comment:
Then all of a sudden you chance writing format? 

1. don't have multiple styles of writing in the same document, very confusing to developers. 
2. preferred to use `attr: rule` syntax with a space